src/components/utils/test/thread_test.cc

```diff
@@ -175,6 +247,175 @@ TEST_F(ThreadTest, StartOneThreadTwice_ExpectTheSameThreadStartedTwice) {
   EXPECT_TRUE(delegate_->check_value());
 }
 
+TEST(ThreadTest2, StartAndJoinForce_Success) {
```

Comment:
@AGaliuzov @AGritsevich Changed the name in be0bf3e46938764116c04f04658be2a7d1b2cfef